xml/release-notes.xml

```diff
@@ -300,6 +300,67 @@
        </step>
        </procedure>
    </sect3>
+   <sect3 xml:id="sec.upgrade.422.shorewall">
+    <title>Shorewall upgrade</title>
+    <para>
+      In &opensuseleap; 42.2, Shorewall was upgrade to last stable 5.1x series.
+      <package>shorewall</package> and <package>shorewall6</package> on upgrade
```

Comment:
https://build.opensuse.org/package/show/openSUSE:Leap:42.3/shorewall & https://build.opensuse.org/package/show/openSUSE:Leap:42.3/shorewall both have Shorewall 4.6 only. Is this still in a staging branch somewhere?

Additionally, the version number at the beginning of the para, "In openSUSE Leap 42.2, Shorewall..." should that be 42.3 instead?